frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx

```diff
@@ -63,7 +65,8 @@ export const FormEndPageContainer = ({
 
   return (
     <Box py={{ base: '1.5rem', md: '2.5rem' }} w="100%">
-      {form?.payments?.enabled ? (
+      {form?.responseMode === FormResponseMode.Encrypt &&
+      form?.payments_field?.enabled ? (
```

Comment:
at this point should we just make a context variable `isPaymentForm`?

Reply:
#5976